README.md

```diff
@@ -53,7 +53,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [NOT AVAILABLE FOR NOW](https:#)
+- [NOW AVAILABLE AT](https://shurichmond.github.io/Super-Portfolio/)
```

Comment:
[OPTIONAL] Nice work adding the live link to your readme😉 this will make a user easily access your website online🤗 But I think we can make the text more descriptive😉 How about `SEE PROJECT LIVE` or `LINK TO LIVE PROJECT`  instead of ` NOW AVAILABLE AT` I think the previous suggestions are more descriptive. you can kindly consider making your text more descriptive🤗